draft-ietf-quic-transport.md

```diff
@@ -3577,6 +3588,10 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
+As future versions of QUIC may support Connection IDs larger than the version 1
+limit, Version Negotiation Packets MAY carry Connection IDs that are longer than
```

Comment:
```suggestion
limit, Version Negotiation Packets could carry Connection IDs that are longer than
```
@martinthomson pointed out that this isn't the spec where we need to allow larger CIDs to happen. This merely needs to note that it could happen.